base/database/utils.go

```diff
@@ -96,3 +97,8 @@ func PluckInt(tx *gorm.DB, columnName string) int {
 	}
 	return val
 }
+
+func IsValidUUID(uuid string) bool {
+	r := regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$")
```

Comment:
This regex can be created in "var" no to be created during each function call. Let's call it e.g. uuidRegex.

~~~golang
var uuidRegex = regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$")

func IsValidUUID(uuid string) bool {
	return uuidRegex.MatchString(uuid)
}

~~~